packages/ra-core/src/types.ts

```diff
@@ -9,10 +9,10 @@ import { AuthActionType } from './auth/types';
 
 export type Identifier = string | number;
 
-export interface RaRecord {
+export type RaRecord = {
```

Comment:
Why?

Reply:
Having it as a type alias seems more appropriate, as we're just describing the shape of a Type, and so it isn't a separate entity itself.
Doesn't make much difference outside of errors talking in terms of the shape, instead of the type `RaRecord`.

I can change it back if you want.